spec/support/github_helper.rb

```diff
@@ -24,34 +24,6 @@ def stub_ci
       def stub_request_source
         Danger::RequestSources::GitHub.new(stub_ci, stub_env)
       end
-
-      def stub_merge_request(fixture, slug, merge_request_id)
```

Comment:
We should try an reintroduce this to make the stubbing less verbose for GitHub though